conn_test.go

```diff
@@ -0,0 +1,20 @@
+package amqp
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestConnProperty(t *testing.T) {
+	c := new(conn)
+	key := "propKey"
+	value := "propValue"
+	err := ConnProperty(key, value)(c)
+	if err != nil {
+		t.Error(fmt.Sprintf("%+v", err))
```

Comment:
I don't know how this sort of thing ended up in the other tests, but it should just be `t.Errorf("%+v", err)`. I need to correct my usages.